### PR TITLE
Add default routes k8s windows

### PR DESCRIPTION
**Pull request by kmurudi**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Comment by nddq** [comment]
is this a part of https://github.com/Azure/azure-container-networking/pull/2634?

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,8 @@ GOOSES   ?= "linux windows" # To override at the cli do: GOOSES="\"darwin bsd\""
 GOARCHES ?= "amd64 arm64" # To override at the cli do: GOARCHES="\"ppc64 mips\""
 ltsc2019  = "10.0.17763.4010"
 ltsc2022  = "10.0.20348.643"
+ltsc2022-KB5035857   = "10.0.26100.1"
+win2025 = "10.0.26100.1"
```
Comment (bohuini) [inline Makefile:26]: How is "10.0.26100.1" got determined?

**Comment by kmurudi** [comment]
> is this a part of #2634?

no this is a fix for aks middleware-windows

**Review by jpayne3506 — CHANGES_REQUESTED** [review]

File: cns/windows.Dockerfile [hunk 2]
```diff
@@ -13,6 +13,9 @@ FROM mcr.microsoft.com/windows/servercore@sha256:6fdf140282a2f809dae9b13fe441635
 # intermediate for win-ltsc2022
 FROM mcr.microsoft.com/windows/servercore@sha256:45952938708fbde6ec0b5b94de68bcdec3f8c838be018536b1e9e5bd95e6b943 as ltsc2022
 
+# FROM mcr.microsoft.com/windows/server/insider:10.0.26080.1 as 
+FROM mcr.microsoft.com/windows/servercore:ltsc2022-KB5035857-amd64 as win2025
```
[the remark this answers is not in the document]
Comment (paulyufan2) [reply cns/windows.Dockerfile:17]: you can probably find -KB5035857 from:
https://support.microsoft.com/en-us/topic/march-12-2024-kb5035857-os-build-20348-2340-a7953024-bae2-4b1a-8fc1-74a17c68203c
KB5014754—Certificate-based authentication changes on Windows domain controllers

File: cns/middlewares/k8sSwiftV2_windows.go [hunk 3]
```diff
@@ -6,7 +6,12 @@ import (
 )
 
 // setRoutes sets the routes for podIPInfo used in SWIFT V2 scenario. This is a no-op as route setting is not applicable for Windows.
-func (k *K8sSWIFTv2Middleware) setRoutes(_ *cns.PodIpInfo) error {
-	logger.Printf("[SWIFTv2Middleware] setRoutes is a no-op on Windows")
+// for AKS L1VH, do not set default route on infraNIC to avoid customer pod reaching all infra vnet services
+// default route is set for secondary interface NIC
+func (k *K8sSWIFTv2Middleware) setRoutes(podIPInfo *cns.PodIpInfo) error {
+	logger.Printf("[SWIFTv2Middleware] setRoutes is a no-op on Windows, only skipDefaultRoutes for InfraNIC")
```
[the remark this answers is not in the document]
Comment (kmurudi) [reply cns/middlewares/k8sSwiftV2_windows.go:12]: we can change this

File: cns/windows.Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (kmurudi) [reply cns/windows.Dockerfile]: we can remove this and ack on paul's comment it is the latest tag published by them which is forward-OS compatible

File: Makefile (hunk 1 above)
Comment (kmurudi) [reply Makefile:26]: This is the platform of l1vh: PS C:\k> systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
OS Name:                       Microsoft Windows Server 2025 Datacenter
OS Version:                    10.0.26100 N/A Build 26100

**Pull request closed without merging**